src/DynamicData/Cache/ObservableCacheEx.cs

```diff
@@ -2194,6 +2194,24 @@ public static IObservable<IReadOnlyCollection<T>> StartWithEmpty<T>(this IObserv
             return sources.Combine(CombineOperator.And);
         }
 
+        /// <summary>
```

Comment:
These functions are needed, as the two functions above them collide now that ISourceList implements them both.  This function is a clarifying function that the compiler can use to remove the ambiguity.  It internally calls the IObservableList version instead of the ICollection version.